src/recommendpolicy/recommendPolicy.go

```diff
@@ -122,43 +129,37 @@ func StopRecommendCronJob() {
 // RecommendPolicyMain generates recommended policies from policy-template GH
 func RecommendPolicyMain() {
 
-	nsNotFilter := cfg.CurrentCfg.ConfigSysPolicy.NsNotFilter
```

Comment:
can we have the `nsNotFilter` string created in this function as a local variable and pass it to `GetHardenPolicy()` rather than having it as a Global Variable

Reply:
added